.github/workflows/cherry-pick-to-frozen.yml

```
@@ -0,0 +1,276 @@
+              issue_number: parseInt(process.env.PR_NUMBER)
+            });
+
+            const hasCPToTrunkLabel = pr.data.labels.some(
+              label => label.name.toLowerCase() === 'cherry pick to trunk'
+            );
+
+            core.setOutput('add_cp_to_trunk_nag', ! hasCPToTrunkLabel);
+
+  handle-successes:
+    needs: [prepare, cherry-pick, cherry-pick-to-trunk-check]
+    if: ${{ always() && needs.cherry-pick.ouputs.status == 'success' && needs.cherry-pick.outputs.cherry_pick_pr_number }}
```

actionlint (1.7.7)

property "ouputs" is not defined in object type {outputs: {cherry_pick_pr_number: string; error_message: string; status: string}; result: string}

(expression)